packages/mds-web-sockets/server.ts

```diff
@@ -27,7 +32,11 @@ export const WebSocketServer = () => {
 
   const clients = new Clients()
 
-  function pushToClients(entity: ENTITY_TYPE, message: string) {
+  function isSupported(entity: string) {
+    return supportedEntities.findIndex(e => e === entity) === -1
```

Comment:
I tried using Array.includes() here to test but it complained that `Argument of type 'string' is not assignable to parameter of type '"event" | "telemetry"'`.  So the default type for supportedEntities was being enforced on the searchParam too.  Switching to findIndex resolves.  Suggestions welcome.

Reply:
Isn't this backwards? Shouldn't it be `!== -1`? `-1` means it wasn't found which would mean it's not supported. Also, could you use `Array.some` instead? 
```ts
return supportedEntities.some(e => e === entity)
```

Reply:
Looks like .some will work, thanks